polling_stations/apps/data_collection/management/commands/import_south_kesteven.py

```diff
@@ -43,16 +48,24 @@ def address_record_to_dict(self, record):
             "10007252178",  # NG323AY -> NG323SB : 4 Main Street, Sudbrook, Grantham, Lincs.
             "100030919971",  # NG322LW -> NG322NH : Corner House, Main Road, Barkston, Grantham, Lincs
             "10007242167",  # NG322DU -> NG322EF : Willow Top Farm, Gonerby Lane, Allington, Grantham, Lincs
-            "10007263149",  # PE91JP -> PE92AP : Flat 1 Caymen House, Foundry Road, Stamford, Lincs
-            "10007263150",  # PE91JP -> PE92AP : Flat 2 Caymen House, Foundry Road, Stamford, Lincs
-            "10007263151",  # PE91JP -> PE92AP : Flat 3 Caymen House, Foundry Road, Stamford, Lincs
-            "10007263152",  # PE91JP -> PE92AP : Flat 4 Caymen House, Foundry Road, Stamford, Lincs
+            "10007246399",  # NG334SP -> NG334SR : The Barn, Porters Lodge Farm, Morkery Lane, Castle Bytham, Grantham, Lincs.
+            "100030904285",  # PE109AE -> PE109EA : Flat Over Bank, R/o 4 North Street, Bourne, Lincs
+            #     "10007263149",  # PE91JP -> PE92AP : Flat 1 Caymen House, Foundry Road, Stamford, Lincs
+            #     "10007263150",  # PE91JP -> PE92AP : Flat 2 Caymen House, Foundry Road, Stamford, Lincs
+            #     "10007263151",  # PE91JP -> PE92AP : Flat 3 Caymen House, Foundry Road, Stamford, Lincs
+            #     "10007263152",  # PE91JP -> PE92AP : Flat 4 Caymen House, Foundry Road, Stamford, Lincs
         ]:
             rec["accept_suggestion"] = True
 
         if uprn in [
             "10007248507",  # NG321LH -> NG318BY : Park Cottage, Park Lane, Denton, Grantham, Lincs
             "200001873982",  # PE91QP -> PE91QL : Hazel Grove, Emlyns Street, Stamford, Lincs
+            "10007257592",  # PE100TT -> PE100TG : Gatehouse Cottage, Fen Road, Dowsby, Bourne, Lincs
+            # Don't know
```

Comment:
Remember if you don't know, its better to do nothing here as opposed to explicitly going with either the addressbase ( `rec["accept_suggestion"] = True` ) or council data ( `rec["accept_suggestion"] = False` ). If we just leave it as a warning, we'll discard the data and say "we don't know", which is the safe option if we can't make a good decision.

Reply:
Noted :-). Now removed